lib/phases/cold-launch.js

```diff
@@ -34,29 +33,15 @@ var ColdLaunch = function(options) {
    * 5. Register to capture performance entries
    * 6. Start runs
    */
-
-  this.getDevice()
-    .then(function() {
-      return phase.device.log.clear();
-    })
-    .then(function() {
-      return phase.device.helpers.restartB2G();
-    })
-    .then(function() {
-      return phase.waitForB2GStart();
-    })
-    .then(function() {
-      return phase.getAppCoordinates();
-    })
-    .then(function() {
-      return phase.prime();
-    })
-    .then(function() {
-      return phase.capture();
-    })
-    .then(function() {
-      return phase.start();
-    });
+  this
+    .getDevice()
+    .then(() => this.device.clearLog())
```

Comment:
For some reason my CLI is dying here now.

rwood@ubuntu:~/cli$ adb devices
List of devices attached 
YT9112BFWN	device

rwood@ubuntu:~/cli$ raptor test coldlaunch --app clock
TypeError: Cannot read property 'then' of undefined
    at new ColdLaunch (/home/rwood/raptor-cli/lib/phases/cold-launch.js:38:5)
    at Object.module.exports.create (/home/rwood/raptor-cli/lib/phases/index.js:27:10)
    at /home/rwood/raptor-cli/lib/index.js:76:24
rwood@ubuntu:~/cli$